trelloapp.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import os
+from flask import Flask, render_template
+import json
+from dotenv import load_dotenv  #to invoke .env file
```

Comment:
If you're running flask via `flask run` you do not need to run `load_dotenv()` manually.